sky/packages/sky/lib/widgets/scrollable.dart

```diff
@@ -478,6 +493,61 @@ class ScrollableList<T> extends FixedHeightScrollable {
   }
 }
 
+class PageableList<T> extends ScrollableList<T> {
+  PageableList({
+    Key key,
+    ScrollDirection scrollDirection,
+    List<T> items,
+    ItemBuilder<T> itemBuilder,
+    double itemSize,
+    EdgeDims padding,
+    this.duration: const Duration(milliseconds: 200),
+    this.curve
```

Comment:
I didn't include a default because scrollTo() does (the duration and curve parameters are used to configure scrollTo()). That said, this API would be a little easier to understand if I specified the default - ease - here.